src/styles/main.scss

```diff
@@ -1,3 +1,66 @@
-body {
-  margin: 0;
+$blockSize: 100px;
+$gapSize: 1px;
+$weekDays:
+  mon 0,
+  tue 1,
+  wed 2,
+  thu 3,
+  fri 4,
+  sat 5,
+  sun 6;
+
+.calendar {
+  position: absolute;
+  display: flex;
+  flex-wrap: wrap;
+
+  padding: $blockSize;
+  width: (7 * $blockSize) + ($gapSize * 6);
+  gap: $gapSize;
+
+  &--month-length {
+    @for $day from 28 through 30 {
+      &--#{$day} .calendar__cell:nth-child(n+#{$day + 1}) {
+        display: none;
+      }
+    }
+  }
+
+  &--start-day {
+    @each $day, $position in $weekDays {
+      &--#{$day} .calendar__cell:first-child {
+        margin-left: $position * ($blockSize + $gapSize);
+      }
+    }
+  }
+
+  &__cell {
+    display: block;
+    box-sizing: border-box;
+    text-align: center;
+
+    width: $blockSize;
+    height: $blockSize;
+    line-height: $blockSize;
+
+    background-color: #eee;
+    border: 1px solid #000;
+    transition: transform 0.5s;
```

Comment:
What about background-color? You change it on hover too.
Also, would be great to create a variable for transition-duration
![image](https://github.com/mate-academy/layout_calendar/assets/96409563/e1f388a1-220b-4d2d-8622-2caa003a25d5)